Dockerfile

```diff
@@ -30,7 +30,7 @@ ENV BUILD_VERSION=${BUILD_VERSION}
 ENV NODE_ENV=${ENV}
 
 # If this is a prod environment, package the code
-RUN if [ "$ENV" != "local" ]; then yarn build; fi
+RUN if [ "$ENV" != "local" ]; then node --run build; fi
```

Comment:
Why drop yarn?

Reply:
Node 22 added the ability to invoke scripts from package.json. I figure it wouldn't be a bad idea to use something built into node over something built into a dated package manager.